Makefile

```diff
@@ -6,30 +6,38 @@ SPHINX_BUILDDIR = docs/_build
 
 help:
 	@echo "  clean                       delete local files"
-	@echo "  docker-build                build the Docker™ image"
-	@echo "  docker-test                 run the tests from within the container"
+	@echo "  build                       build the Docker™ image"
+	@echo "  test                        run the tests from within the container"
 	@echo "  lambda.zip                  build lambda.zip from within the container"
+	@echo "  lintcheck                   run lint checking (i.e. flake8)"
 	@echo "  upload-to-s3                upload lambda.zip to AWS"
 	@echo "  docs                        build the project docs"
+	@echo "  shell                       enter a bash shell with volume mount"
 
-virtualenv: $(PYTHON)
-$(PYTHON):
-	$(VIRTUALENV) $(VENV) --python=python3.6
+#
+# virtualenv: $(PYTHON)
+# $(PYTHON):
+# 	$(VIRTUALENV) $(VENV) --python=python3.6
+
+.docker-build:
+	make build
```

Comment:
maybe this could depend on some source like setup.py to avoid having to run `make clean` manually